spec/cedar.spec.js

```diff
@@ -172,7 +173,8 @@ describe('Cedar', function () {
 
   describe('mapping names', function () {
     it("should append _SUM for count", function() {
```

Comment:
Rathr than just changing this test so it passes, I would mark this test skipped w/ xit() - that way it will stand out.